compio-signal/src/unix.rs

```diff
@@ -27,9 +27,10 @@ impl Pipe {
     pub fn new() -> io::Result<Self> {
         let (receiver, sender) = os_pipe::pipe()?;
 
-        std::thread::spawn(move || {
+        compio_runtime::spawn(compio_runtime::spawn_blocking(move || {
```

Comment:
This will permanently try one usable thread from the pool. Is that desirable?

Reply:
TBH I'm not sure about that.

Reply:
I think we should keep the plain std thread and just add that doc comment.